Sources/NIOTransportServices/NIOTSEventLoopGroup.swift

```diff
@@ -85,4 +85,34 @@ public final class NIOTSEventLoopGroup: EventLoopGroup {
         return EventLoopIterator(self.eventLoops)
     }
 }
+
+/// A TLS provider to bootstrap TLS-enabled connections with `NIOClientTCPBootstrap`.
+///
+/// Example:
+///
+///     // Creating the "universal bootstrap" with the `NIOTSClientTLSProvider`.
+///     let tlsProvider = NIOTSClientTLSProvider()
+///     let bootstrap = NIOClientTCPBootstrap(NIOTSConnectionBootstrap(group: group), tls: tlsProvider)
+///
+///     // Bootstrapping a connection using the "universal bootstrapping mechanism"
+///     let connection = bootstrap.enableTLS()
+///                          .connect(to: "example.com")
```

Comment:
Does `connect(to: String)` exist?!

Reply:
whoops, no. `host:port:`. great catch, thanks, fixed